### ISSUE TITLE
Ensure normalization and usage

### ISSUE THREAD

**Issue by gagern** [CindyJS/CindyJS#390]
Currently some operations (like `ConicBy4p1l`) return matrices which are *not* normalized (using `normalizeMax`) and do *not* carry appropriate usage information. The former could lead to avoidable overflow while the latter may cause problems when `draw` decides whether to draw a point or a line. So these requirements should be checked systematically for all operations, and fixed where broken.

**Comment by elkins0** [comment]
So you are proposing to move that functionality up and out of `geoOps.SelectConic.updatePosition`?
```javascript
geoOps.SelectConic.updatePosition = function(el) {
    var set = csgeo.csnames[(el.args[0])];
    el.matrix = set.results[el.param];
    el.matrix = List.normalizeMax(el.matrix);
    el.matrix = General.withUsage(el.matrix, "Conic");
};
```

**Comment by gagern** [comment]
@elkins0: Good point, I had missed the fact that we had this behavior there. On the other hand, we definitely should normalize before computing the adjoint, as some operations do. The tracing might be more stable with normalized matrices as well. So I think there is still work to be done here, but now it's less clear-cut what needs to be changed and what can remain as is.
